### PR TITLE
Add permissions boundary to the docs (we've added it to the bucket already)

### PR DESCRIPTION
**Pull request by dnarain-astro**

### CODE REVIEW

**Comment by dnarain-astro** [comment]
Can you please approve this latest edit to the private preview for cross account roles?

**Review by kwcanuck — COMMENTED** [review]
@dnarain-astro thanks for taking this on. I made minor edits to your content. When time permits, please review the changes. If you have questions, let me know. If you're happy with the content, you can merge, or you can notify me when you've completed your review and I'll merge the changes.

**Review by kwcanuck — APPROVED** [review]

File: astro/cross-account-role-setup.md [hunk 1]
```diff
@@ -187,23 +189,23 @@ Use the external ID to create a cross-account IAM role for Astro. Astronomer rec
...
-#### Monitor the cross-account role for changes (optional)
+#### Monitor the policies for changes (optional)
 
-You can use CloudTrail to monitor changes to policies within the cross-account role.  Access to CloudTrail has been limited to prevent the accidental modification or deletion of CloudTrail logs by Astronomer support. The following table lists the events that you should monitor. 
+You can use CloudTrail to monitor changes to Astro policies.  Access to CloudTrail has been limited to prevent the accidental modification or deletion of CloudTrail logs by Astronomer support. The following table lists the events that you should monitor. 
 
 | Event Names                              | Resource                                                         |
 | ---------------------------------------- | ---------------------------------------------------------------- |
 | `AttachRolePolicy , DetachRolePolicy`    | `roleName = astronomer-remote-management`                        |
 | `SetPolicyVersion , CreatePolicyVersion` | `policyArn = "arn:aws:iam::*:policy/AstronomerCrossAccountRole"` |
```
Comment (dnarain-astro) [inline astro/cross-account-role-setup.md:203]: Crap, I forgot to delete CreatePolicyVersion from this list.

**Review by dnarain-astro — COMMENTED** [review]
Thanks

**Comment by dnarain-astro** [comment]
Merged.

**Pull request merged**